assets/js/admwpp/shortcode.js

```diff
@@ -183,8 +183,8 @@
           } else {
             $('tbody', parent).append(response.html);
 
-            if (response.hasNextPage) {
-              button.data('page', response.hasNextPage + 1);
+            if (response.bundledLps.hasNextPage) {
+              button.data('page', response.bundledLps.currentPage + 1);
```

Comment:
should we add both conditions?
if response.bundledLps.hasNextPage else if response.hasNextPage ?

Reply:
response.bundledLps will always exist response.hasNextPage s not based on the PHP action attached to the ajax hook.
`$response = array(
                'bundledLps' => $bundledLps,
                'params' => $params,
                'html' => $html,
            );
            echo json_encode($response);
            wp_die();`